src/matrices/sqmatrix.c

```diff
@@ -323,6 +324,7 @@ void sqmatrix_sqrt(sqmatrix Usqrt, sqmatrix U, sqmatrix W)
      CHECK(Usqrt.p == U.p && U.p == W.p, "matrices not conformant");
 
      CHK_MALLOC(eigenvals, real, U.p);
+     memset(eigenvals, 0, sizeof(real) * U.p);
```

Comment:
Shouldn't this be done in `sqmatrix_eigensolve`, i.e. won't similar warnings come up on other calls to `sqmatrix_eigensolve` if Msan doesn't understand LAPACK output variables?

Also should have a comment that that this initialization is not actually necessary, but is just to silence Msan.

Reply:
I agree that this is an incomplete solution to silence the MSan based on the unit tests provided from MEEP. It would give additional warnings if other similar functions are hit, e.g. `sqmatrix_eigensolve`. It is quite hard to find all the MSan warnings without additional testing cases. Now maybe it is not necessary to silence warnings of use-of-uninitialized-value if you think `calloc` is not a good solution. Please feel free to close this one and the other [one](https://github.com/NanoComp/harminv/pull/8#issuecomment-777183572) if you agree to leave as it is. Thanks.